src/testdrive/src/util/postgres.rs

```diff
@@ -50,8 +51,11 @@ pub fn config_url(config: &Config) -> Result<Url, anyhow::Error> {
 
 pub async fn postgres_client(
     url: &str,
+    default_timeout: Duration,
 ) -> Result<(Client, JoinHandle<Result<(), tokio_postgres::Error>>), anyhow::Error> {
-    let tls = make_tls(&Config::from_str(url)?)?;
+    let pgconfig = &mut Config::from_str(url)?;
+    pgconfig.connect_timeout(default_timeout);
+    let tls = make_tls(pgconfig)?;
```

Comment:
I always thought `default_timeout` was used to determine how long testdrive would retry a single test case. This looks more like how long it will attempt to connect to the actual database.

Reply:
We want to have no hard-coded timeouts in testdrive anywhere (as no hard-coded default is ever appropriate for all mzcompose workflows), so they all must be configurable. In order to avoid having many different configurable timeout options, we use default_timeout to govern all of them.

Reply:
I misunderstood before. I thought you were missing the timeout during connection, let me try to add it for queries too.